x-pack/plugins/security_solution/public/entity_analytics/components/risk_engine_privileges_callout/risk_engine_privileges_callout.tsx

```diff
@@ -25,6 +26,10 @@ export const RiskEnginePrivilegesCallOut: React.FC<{
   };
 
   return (
-    message && <CallOutSwitcher namespace="entity_analytics" condition={true} message={message} />
+    <>
+      <CallOut message={message} showDismissButton={false} />
+      <EuiSpacer size="l" />
```

Comment:
We should not assume all places this component is used will need this specific space. We should let every specific page add the space they need.

Reply:
Yeah, I thought about that, but I copied it from the `CalloutSwitcher`, which itself makes that assumption.
Since we use that component for most other callouts, I wanted to keep this consistent.